weaviate/collections/data.py

```diff
@@ -227,11 +243,15 @@ def __apply_context_to_params_and_object(
             params["consistency_level"] = self._consistency_level
         return params, obj
 
-    def _serialize(self, props: Properties, refs: Optional[WeaviateReferences]) -> Dict[str, Any]:
-        return {
-            **({key: val._to_beacons() for key, val in refs.items()} if refs is not None else {}),
-            **{key: self.__serialize_primitive(val) for key, val in props.items()},
-        }
+    def _serialize_props(self, props: Properties) -> Dict[str, Any]:
+        return (
+            {key: self.__serialize_primitive(val) for key, val in props.items()}
+            if props is not None
```

Comment:
in all the calls I've seen you do
```
props = {}
if properties is not None:
    props = _serialize_props(properties)
```

I think it makes sense to handle the None case in here and just do `props = _serialize_props(properties)` without the none handling at the caller side

Reply:
same for refs

Reply:
I made this change because the typing was annoying to deal with when `Optional`s are present. I will check to see if I can refactor it another way

Reply:
then just leave out the `if props is not None` in here?

Reply:
Oh yeh, that's been left in there by accident, thanks for spotting!